src/main/flight/pid.c

```diff
@@ -344,6 +344,12 @@ static void pidApplyRateController(const pidProfile_t *pidProfile, pidState_t *p
         pidState->errorGyroIfLimit = ABS(pidState->errorGyroIf);
     }
 
+#ifdef USE_SERVOS
```

Comment:
Yes, and I use CJMCU as my development board for NRF24L01 stuff - since it through-holes for SPI that a NRF24 board can plug directly into.

It's quite fun as well, I can fly it with my CX10 controller (or my JJRC H20 controller).